bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/igrins2/Igrins2.scala

```diff
@@ -179,7 +179,7 @@ final class Igrins2 extends ParallacticAngleSupportInst(Igrins2.SP_TYPE)
     Duration.ofSeconds(Igrins2.SetupTime.toSeconds.toLong)
 
   override def getReacquisitionTime(config: Config): Duration =
-    Duration.ofSeconds(Igrins2.SetupTime.toSeconds.toLong)
+    Duration.ofMinutes(5)
```

Comment:
Are you sure about this. This was recently changed as per:
https://noirlab.atlassian.net/browse/REL-4531